custreamz_notebooks/random_haproxy_data_generator.ipynb

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
I like the addition of the notebook for the user to generate Random HAProxy data. Nice touch

Reply:
It was your idea, thanks for this!